### DISCUSSION TITLE
Frontend Help

### DISCUSSION THREAD

**Discussion by ivavay** [mdwiltfong/secret-santa#38] [Q&A]
Hi! I'm finally ready to get back into it. What is there on the frontend part that I can help with? I realized I'm a bit rusty on the more intermediate concepts in react, like state management, so I could look into it more and get more practice maybe.

**Comment by mdwiltfong** [comment]
Nice! So glad you're up for more! Right now we are waiting on the PR for this issue to be solved: https://github.com/mdwiltfong/secret-santa/issues/20

With redux we can finally start implementing state management in the front end, and start authenticating/authorizing users.
